.ruby-version

```diff
@@ -0,0 +1 @@
+2.7.1
```

Comment:
I like you specify the ruby version
Just curious 😄 
What tool are you using for handling multiple ruby-versions in your env?

Reply:
I used rbenv